webpack.config.dev.js

```diff
@@ -11,8 +11,8 @@ import writeAssets from './src/server/write-assets'
 
 const assetsPath = path.resolve(__dirname, 'public')
 
-const WEBPACK_HOST = process.env.HOST || 'localhost'
-const WEBPACK_PORT = parseInt(process.env.PORT) + 1 || 2031
+const WEBPACK_HOST = process.env.WEBPACK_HOST || '0.0.0.0'
```

Comment:
Thanks for this test!
I have not been able to understand the use case for changing `WEBPACK_HOST` and have thus probably been too lax on testing.

Can I ask how you tried to set this value? I feel it would probably be a consequence of https://github.com/openfisca/legislation-explorer/pull/165#discussion_r206553768.

Reply:
I ended up removing the ability to set up `WEBPACK_HOST` and `WEBPACK_PORT` independently from `HOST` and `PORT`, it adds complexity and I can't see a proper use case for them. Please let me know if you had a use for it :wink: